docs/install.md

```diff
@@ -261,5 +261,32 @@ sudo udevadm control --reload
 
 Then, you will need to unplug and replug the key for the rule to trigger.
 
+## Verifying the installation
+
 To test whether the installation was successful, visit a
 [demo website](https://webauthn.io/) and try to register and login.
+
+### Troubleshooting (Linux only)
```

Comment:
I would put it at the same level as testing the website and divide into subsections per OS:
```
## Troubleshooting

### Linux

### OS X
```

Reply:
I don't have anything specific for OSX so leaving this section out for now, but sure I can rename the sections.

Reply:
Sent https://github.com/google/OpenSK/pull/41 to push to this branch. GitHub's web UI is quite confusing...